go.mod

```diff
@@ -14,8 +14,13 @@ require (
 	github.com/ferranbt/fastssz v0.1.3
 	github.com/gdamore/tcell/v2 v2.6.0
 	github.com/glendc/go-external-ip v0.1.0
+	github.com/go-openapi/errors v0.21.0
```

Comment:
FYI: I needed the openapi dependencies for the client to call alertmanager. The other version bumps were done by go. I didn't notice any problems with those bumps but they were really just done by the `go mod...` command (get or tidy, I forget which).